api/data_explorer/__main__.py

```diff
@@ -165,7 +166,21 @@ def _process_bigquery():
         sample_file_columns = bigquery_config.get('sample_file_columns', {})
         table_names.sort()
 
-    app.app.config['TABLE_NAMES'] = table_names
+    table_names_dict = OrderedDict()
+    for full_table_name in table_names:
+        splits = full_table_name.split('.')
```

Comment:
(Sorry, I see you copied this code -- I have some comments on it)

Project id can have `.` in it (google.com). So use rsplit to make 3 splits.

Reply:
Done.